docs/application/dotnet/guides/multimedia/media-content.md

```diff
@@ -336,11 +338,11 @@ using (var mediaDataReader = folderCmd.SelectMedia(folder.Id))
 ```
 
 
-## Retrieving Media Information
+## Retrieve media information
 
-To access media item information:
+To access media item information, follow the steps below:
 
-1.  The following example filters media items so that only image and video items are included in the result. The filter is case-insensitive, and the results are sorted in descending order by item display name. For more information on the filter properties, see [Setting up a Filter](#setting-up-a-filter).
+1.  The following example filters media items so that only image and video items are included in the result. The filter is case-insensitive, and the results are sorted in descending order by item display name. For more information on the filter properties, see [Set up a Filter](#set-up-a-filter).
```

Comment:
```suggestion
1.  The following example filters media items so that only image and video items are included in the result. The filter is case-insensitive, and the results are sorted in descending order by item display name. For more information on the filter properties, see [set up a filter](#set-up-a-filter).
```